tests/test_init.py

```diff
@@ -53,3 +60,99 @@ def test_parse_bad_patchwork_url(self):
             "Can't parse patchwork url: '{}'".format(patchwork_url)
             in context.exception
         )
+
+
+class KBuilderTest(unittest.TestCase):
+    """Test cases for skt.kbuilder class"""
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+        self.tmpconfig = tempfile.NamedTemporaryFile()
+        self.kbuilder = skt.kbuilder(self.tmpdir, self.tmpconfig.name)
+        self.m_popen = Mock()
+        self.m_popen.returncode = 0
+        self.ctx_popen = mock.patch('subprocess.Popen',
+                                    Mock(return_value=self.m_popen))
+        self.ctx_check_call = mock.patch('subprocess.check_call', Mock())
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+
+    def test_clean_mktgz(self):
+        """Check if `mrproper` target is called when clean is True"""
```

Comment:
This checks much more than just the `mrproper` target being "made". Please remove the rest of the tests from here, and please also only check that there is a command which has `make` as the first argument and `mrproper` as one of the following arguments. Otherwise this test is going to break often (including when you rebase now).

Reply:
Ok, done